api/src/digest/cron.ts

```diff
@@ -67,11 +70,14 @@ export class DigestCron {
     const projectsFromDataFolder = await this.dataService.listProjects();
 
     for (const project of projectsFromDataFolder) {
-      const [{ id: projectId }] = await this.projectsRepository.upsert({
+      const projectEntity = {
```

Comment:
lets apply this suggestion to other entities as well:

```suggestion
      const projectEntity: ProjectRow = {
```